### PR TITLE
Use only 'https' in CSP.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian pr updated to include overrides for local dev.

File: remo/settings/local.py-dist [hunk 1]
```diff
@@ -125,3 +125,27 @@ SESSION_COOKIE_SECURE=False
...
+)
+CSP_FONT_SRC = base.CSP_FONT_SRC + (
+    'http://*.mozilla.net',
+    'http://*.mozilla.org',
+    'http://mozorg.cdn.mozilla.net',
+)
+CSP_IMG_SRC = base.CSP_IMG_SRC + (
+    'http://*.mozilla.net',
+    'http://*.mozilla.org',
+    'http://*mapbox.com',
```
Comment (johngian) [inline remo/settings/local.py-dist]: Not valid csp rule.

File: remo/settings/local.py-dist [hunk 2]
```diff
@@ -125,3 +125,27 @@ SESSION_COOKIE_SECURE=False
...
+)
+CSP_SCRIPT_SRC = base.CSP_SCRIPT_SRC + (
+    'http://*.mozilla.net',
+    'http://*.mozilla.org',
+    'http://*mapbox.com',
```
Comment (johngian) [inline remo/settings/local.py-dist]: Same here.

**Comment by johngian** [comment]
Please fix this error:
```
Content Security Policy: Couldn't parse invalid host http://*mapbox.com
```

**Comment by johngian** [comment]
r+

**Comment by akatsoulas** [comment]
done. thanks!

**Pull request merged**